### PR TITLE
Add description how to use AudioPlayers with audio_service

### PR DESCRIPTION
**Pull request by beerline**
Since it is a lots of issues about how to use AudioPlayers with audio_service, i decided write article about it and add link to it, to avoid future questions

### CODE REVIEW

**Review by luanpotter — APPROVED** [review]
Awesome! There is also a section on the feature-parity table where we mention audio_service, we should probably add it there too!

**Comment by beerline** [comment]
@luanpotter, thank you for review! 
I added description to feature-parity table

File: feature_parity_table.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This is not the best home for them though. We are working with @ryanheise to eve
  * create a new package, audioplayers_notifications for it
  * merge this code into the existing audio_service package
 
-audio_service is already a package that provides much more advanced notification/lock screen controls.
+audio_service is already a package that provides much more advanced notification/lock screen controls. Please flow this [example](https://denis-korovitskii.medium.com/flutter-demo-audioplayers-on-background-via-audio-service-c95d65c90ae1) to implement all AudioPlayers features with and audio_service.
```
Comment (erickzanardo) [inline feature_parity_table.md:60]:
```suggestion
audio_service is already a package that provides much more advanced notification/lock screen controls. Please follow this [example](https://denis-korovitskii.medium.com/flutter-demo-audioplayers-on-background-via-audio-service-c95d65c90ae1) to implement all AudioPlayers features with and audio_service.
```

**Review by erickzanardo — APPROVED** [review]
Small typo, but LGTM

**Pull request merged**